bin/start-chronos.bash

```diff
@@ -1,27 +1,39 @@
 #!/bin/bash
-set -o errexit -o nounset -o pipefail
+set -o errexit -o pipefail
 
-chronos_home="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && cd .. && pwd -P )"
-echo "Chronos home set to $chronos_home"
-export JAVA_LIBRARY_PATH="/usr/local/lib:/lib:/usr/lib"
-export LD_LIBRARY_PATH="${LD_LIBRARY_PATH:-/lib}"
-export LD_LIBRARY_PATH="$JAVA_LIBRARY_PATH:$LD_LIBRARY_PATH"
```

Comment:
@elingg `$@` will success even is you pass no arguments to the script, though at that point `[ -z "$1" ]` will catch that, so the whole thing is moot.

Example:
```
#!/bin/bash
set -o nounset -o errexit -o pipefail
echo "Arguments: ${@}"     # will work if you pass no arguments with "-o nounset"
```